src/server.cc

```diff
@@ -608,6 +609,19 @@ void Server::GetRocksDBInfo(std::string *info) {
     db->GetIntProperty(cf_handle, "rocksdb.estimate-table-readers-mem", &index_and_filter_cache_usage);
     string_stream << "index_and_filter_cache_usage:[" << cf_handle->GetName() << "]:" << index_and_filter_cache_usage
                   << "\r\n";
+    db->GetMapProperty(cf_handle, rocksdb::DB::Properties::kCFStats, &cf_stats_map);
+    string_stream << "level0_file_limit_slowdown:[" << cf_handle->GetName() << "]:"
+                  << cf_stats_map["io_stalls.level0_slowdown"] << "\r\n";
+    string_stream << "level0_file_limit_stop:[" << cf_handle->GetName() << "]:"
+                  << cf_stats_map["io_stalls.level0_numfiles"] << "\r\n";
+    string_stream << "pending_compaction_bytes_slowdown:[" << cf_handle->GetName() << "]:"
+                  << cf_stats_map["io_stalls.slowdown_for_pending_compaction_bytes"] << "\r\n";
+    string_stream << "pending_compaction_bytes_stop:[" << cf_handle->GetName() << "]:"
+                  << cf_stats_map["io_stalls.stop_for_pending_compaction_bytes"] << "\r\n";
+    string_stream << "memtable_count_limit_slowdown:[" << cf_handle->GetName() << "]:"
+                  << cf_stats_map["io_stalls.memtable_slowdown"] << "\r\n";
+    string_stream << "memtable_count_limit_slowdown:[" << cf_handle->GetName() << "]:"
+                  << cf_stats_map["io_stalls.memtable_compaction"] << "\r\n";
```

Comment:
```suggestion
    string_stream << "memtable_count_limit_stop:[" << cf_handle->GetName() << "]:"
                  << cf_stats_map["io_stalls.memtable_compaction"] << "\r\n";
```

Reply:
I don't think it's necessary to add `io_stalls.total_slowdown` and `io_stalls.total_stop`. Since this value is equal to the sum of the listed three, the user can simply add them through the monitoring system.

If we all add it, we'll have too much information.